src/lib/apiTypesV2/BatchQuery.cpp

```diff
@@ -53,38 +53,6 @@ BatchQuery::~BatchQuery()
 
 /* ****************************************************************************
 *
-* BatchQuery::check - 
-*/
-std::string BatchQuery::check(ConnectionInfo* ciP, RequestType requestType)
```

Comment:
Probably this check() method (and a similar one in BatchUpdate) was designed to to the whole validation in one shot... However, by the way the validation code has evolved this is not actually used (maybe it was never used?).

Thus, let's remove now. If in the future we want doing this in this way, it makes more sense to re-implement at that moment that keep a dead piece of code which original purpose has lost in the mist of the past.